core/src/main/java/org/mskcc/cbio/oncokb/model/IndicatorQueryResp.java

```diff
@@ -1,42 +1,76 @@
 package org.mskcc.cbio.oncokb.model;
 
-
 import org.mskcc.cbio.oncokb.apiModels.Implication;
 import org.mskcc.cbio.oncokb.apiModels.MutationEffectResp;
 
+import io.swagger.annotations.ApiModelProperty;
+
 import java.util.ArrayList;
 import java.util.List;
 
-
-/**
- * TumorType generated by hbm2java
- */
-
 public class IndicatorQueryResp implements java.io.Serializable {
     private Query query;
-    private Boolean geneExist;
-    private Boolean variantExist;
-    private Boolean alleleExist;
+
+    @ApiModelProperty(value = "Indicates whether the gene is an OncoKB cancer gene")
```

Comment:
Is correct to say, "Indicates whether the gene is contained in OncoKB"?

Reply:
Probably, "Indicates whether the gene is curated" - what do you think? I think all three is trying to convey the same thing, but we should pick one that makes sense to us.

Reply:
Updated to "Indicated whether the gene is curated by OncoKB"